gluetool/glue.py

```diff
@@ -202,6 +203,36 @@ def func_wrapper(obj, *fargs, **fkwargs):
     return wrap
 
 
+def shared_function(*args, **kwargs):
```

Comment:
Unfortunatelly, the syntax you propose is not possible in Python 2, it won't let you throw a keyword argument between `*args` and `**kwargs`. It is possible in Python 3, though. I can omit `**kwargs` but then the signature must be `name=None, *args` and in that case Python would name the given function `name`, instead of adding it to `args[0]`, and that was very messy :(

Reply:
ok [RESOLVED]